Makefile

```diff
@@ -54,15 +50,18 @@ sync-run: generate
 
 .PHONY: sync-run
 
-monitoring: generate
+monitoring:
 	go build -ldflags ${LDFLAGS} ./cmd/$@
+.PHONY: monitoring
 
 monitoring-run: monitoring
 	./hack/monitoring.sh
 
 monitoring-stop:
 	./hack/monitoring.sh clean
 
+.PHONY: monitoring-run monitoring-stop
```

Comment:
the resultant Makefile .PHONY's look a bit of a mess. can you either place them after the target, in sections or at the top?

Reply:
+1, I'd vote for a single .PHONY target at the top.

Reply:
can do this as a follow-up - what I'm even more interested in is reducing the Makefile down even further...